grails-app/i18n/messages.properties

```diff
@@ -1577,7 +1577,7 @@ order.pending.label=Pending
 order.pendingOrderLog.label=Pending orders
 order.placeOrder.label=Place order
 order.print.label=Print order
-order.filterByName.label=Filter by product name
+order.filterByNameOrCode.label=Filter by product name or code
```

Comment:
Just so we don't have to keep changing it, let's call this filterByProduct.

Reply:
honestly, I made this change on purpose to see, if our crowdin's PR will be triggered, but ok, I'll revert that